algorithms/CSharp/src/Graph/dijkstra.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Priority_Queue;

```

Comment:
This library is needed? because if found this for [C#](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.priorityqueue-2?view=net-6.0&viewFallbackFrom=netstandard-2.1)

Reply:
@ming-tsai it seems to me that this is available for `.Net 6.0`. Couldn't use it in `Visual Studio 2019` or [.Net Finddle](https://dotnetfiddle.net/). Any idea how to use it? If I can get a compiler than I could replace the library

Reply:
Why do we need this library?

Reply:
I couldn't use `PriorityQueue` from `System.Collections.Generic` that is why I used the Priority Queue from this library. If you can help me with using `PriorityQueue` from `System.Collections.Generic` then I can remove this

Reply:
> @ming-tsai  I couldn't use `PriorityQueue` from `System.Collections.Generic` that is why I used the Priority Queue from this library. If you can help me with using `PriorityQueue` from `System.Collections.Generic` then I can remove this

Could you tell me why you need `PriorityQueue`?

Reply:
> > @ming-tsai  I couldn't use `PriorityQueue` from `System.Collections.Generic` that is why I used the Priority Queue from this library. If you can help me with using `PriorityQueue` from `System.Collections.Generic` then I can remove this
> 
> Could you tell me why you need `PriorityQueue`?

I need a min-heap to implement the algorithm. That is why using a `PriorityQueue`

Reply:
Ok, I will check it out